examples/cox.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+"""Cox process model using mean-field variational inference.
+   Ref: Miller et al, 2014.
```

Comment:
I added the origin paper from Cox himself...Cox, 1955. I do think we need to keep Miller et al as reference since that might be quicker to look at than the origin Cox work for someone who wants to quickly work out the details of the model. In Miller et al they do Elliptical slice sampling...Adding their reference was to give a reference to the model not the algorithm...